src/components/atoms/Countdown/Countdown.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react'
+import './Countdown.scss'
+
+const diff = (a, b) => {
+  const distance = a - b
+
+  return {
+    day: Math.floor(distance / (1000 * 60 * 60 * 24)),
+    hour: Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60)),
+    min: Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60)),
+    sec: Math.floor((distance % (1000 * 60)) / 1000),
+  }
+}
+
+const Countdown = ({
+  endTime = new Date().toISOString(),
+  className = '',
+  ...props
+}) => {
+  const [now, setDate] = useState(+new Date())
```

Comment:
Replace occurrences of `+new Date()` with `Date.now()`. Same result, less computations :)

Reply:
fixed @korokd